### ISSUE TITLE
Tests?

### ISSUE THREAD

**Issue by AdrieanKhisbe** [concon121/serverless-plugin-nested-stacks#6]
The package.json mention some tests?
Notably using jasmine. 

Are they unversionned tests, or this is just an artefact of scaffolding the project?

Setting up a CI could be interesting. I can do it if you want @concon121

**Comment by concon121** [comment]
As you suspected, that is just a result of me copying a pasting from another package.json 😨 

It would definitely be worth adding some sensible unit tests to this project given people have started using it!

**Comment by AdrieanKhisbe** [comment]
I can help you on that if you want, 
and set up CI with travis, and lint with eslint.
What do you think about this?

**Comment by concon121** [comment]
Yeah if you have the time feel free 😄 any contributions to make thing better are always welcome!

**Comment by AdrieanKhisbe** [comment]
I set up the CI and the coverage. This will help to write more tests

**Comment by concon121** [comment]
Awesome work, much appreciated!
